src/Typography/Font.php

```diff
@@ -11,6 +11,8 @@ class Font implements FontInterface
     protected float $size = 12;
     protected float $angle = 0;
     protected mixed $color = '000000';
+    protected mixed $strokeWidth = 1;
+    protected mixed $strokeColor = '000000';
```

Comment:
We already have black as font color. I suggest to set `ffffff` as default for stroke color.

Reply:
The font system has been heavily rewritten by me in https://github.com/Intervention/image/pull/1292. I suggest that you build your PR on the state of https://github.com/Intervention/image/pull/1292,

Please I sincerely don't know how to do this...

Reply:
Also I am not able to test imagegick so I couldn't update it to avoid bug and the whole code crashing

Reply:
I would suggest to merge [feature/text-wrapping](https://github.com/Intervention/image/tree/feature/text-wrapping) into this (your) feature branch and fix possible conflicts.

Reply:
Text outline with Imagick is definitely possible.

```php
$draw = new ImagickDraw();
$draw->setStrokeColor('#000');
$draw->setStrokeWidth(4);
$draw->setStrokeAntialias(true);
```

https://www.php.net/manual/en/imagickdraw.setstrokecolor.php
https://www.php.net/manual/en/imagickdraw.setstrokewidth.php
https://www.php.net/manual/en/imagickdraw.setstrokeantialias.php

And it's much easier than with GD. I would ask you to integrate it in your PR.